### PR TITLE
Fix Operators Issue

### PR DESCRIPTION
**Pull request by DanielMwendwa**

### CODE REVIEW

File: public/graph.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ class GraphController {
         return newDate;
     }
 
-    static updateGraphs(data) {
+    static updateGraphs(data, project) {
```
Comment (IsaackMwenda) [inline public/graph.js]:
```suggestion
    static updateGraphs(data, projectName) {
```

**Review by IsaackMwenda — COMMENTED** [review]
This lgtm! as a temporary fix. We can deploy it as a branch but don't merge it to master.

File: public/graph.js [hunk 2]
```diff
@@ -99,17 +80,12 @@ class GraphController {
...
-                    nationlink_sent: d3.sum(v, d => d.nationlink_sent),
-                    somnet_sent: d3.sum(v, d => d.somnet_sent),
-                    somtel_sent: d3.sum(v, d => d.somtel_sent),
-                    telesom_sent: d3.sum(v, d => d.telesom_sent),
-                    golis_sent: d3.sum(v, d => d.golis_sent),
-                    total_sent: d3.sum(v, d => d.total_sent)
-                };
+                let sentData = {};
+                operators.forEach(op => {
+                    sentData[`${op}_sent`] = d3.sum(v, d => d[`${op}_sent`]);
```
Comment (IsaackMwenda) [inline public/graph.js]:
```suggestion
                    sentData[`${operator}_sent`] = d3.sum(v, d => d[`${operator}_sent`]);
```

File: public/graph.js [hunk 3]
```diff
@@ -69,17 +55,12 @@ class GraphController {
...
-                    nationlink_received: d3.sum(v, d => d.nationlink_received),
-                    somnet_received: d3.sum(v, d => d.somnet_received),
-                    somtel_received: d3.sum(v, d => d.somtel_received),
-                    telesom_received: d3.sum(v, d => d.telesom_received),
-                    golis_received: d3.sum(v, d => d.golis_received),
-                    total_received: d3.sum(v, d => d.total_received)
-                };
+                let receivedData = {};
+                operators.forEach(op => {
+                    receivedData[`${op}_received`] = d3.sum(v, d => d[`${op}_received`]);
```
Comment (IsaackMwenda) [inline public/graph.js]:
```suggestion
                    receivedData[`${operator}_received`] = d3.sum(v, d => d[`${operator}_received`]);
```

File: public/graph.js [hunk 4]
```diff
@@ -26,27 +26,13 @@ class GraphController {
...
-            d.somtel_sent = +d.operators["somtel"]["sent"];
-            d.telesom_sent = +d.operators["telesom"]["sent"];
-            d.telegram_sent = +d.operators["telegram"]["sent"];
-            d.NC_sent = +d.operators["NC"]["sent"];
             Object.keys(d.operators)
                 .sort()
                 .forEach(key => {
                     if (!(key in operators)) {
                         operators.add(key);
+                        d[`${key}_received`] = +d.operators[key]["received"];
```
Comment (IsaackMwenda) [inline public/graph.js]: renaming key to operator will make this more readable.

**Review by IsaackMwenda — APPROVED** [review]
Thanks @DanielMwendwa

File: public/graph.js (hunk 4 above)
Comment (DanielMwendwa) [reply public/graph.js]: Noted

**Pull request merged**